apiv2/internal/server/os.go

```diff
@@ -37,25 +37,25 @@ func toInvOSResource(osResource *osv1.OperatingSystemResource) (*inv_osv1.Operat
 		return &inv_osv1.OperatingSystemResource{}, nil
 	}
 	invOSResource := &inv_osv1.OperatingSystemResource{
-		Name:              osResource.GetName(),
-		Architecture:      osResource.GetArchitecture(),
-		KernelCommand:     osResource.GetKernelCommand(),
-		UpdateSources:     osResource.GetUpdateSources(),
-		ImageUrl:          osResource.GetImageUrl(),
-		ImageId:           osResource.GetImageId(),
-		Sha256:            osResource.GetSha256(),
-		ProfileName:       osResource.GetProfileName(),
-		ProfileVersion:    osResource.GetProfileVersion(),
-		InstalledPackages: osResource.GetInstalledPackages(),
-		SecurityFeature:   inv_osv1.SecurityFeature(osResource.GetSecurityFeature()),
-		OsType:            inv_osv1.OsType(osResource.GetOsType()),
-		OsProvider:        inv_osv1.OsProviderKind(osResource.GetOsProvider()),
-		Description:       osResource.GetDescription(),
-		Metadata:          osResource.GetMetadata(),
-		ExistingCvesUrl:   osResource.GetExistingCvesUrl(),
-		ExistingCves:      osResource.GetExistingCves(),
-		FixedCvesUrl:      osResource.GetFixedCvesUrl(),
-		FixedCves:         osResource.GetFixedCves(),
+		Name:                 osResource.GetName(),
+		Architecture:         osResource.GetArchitecture(),
+		KernelCommand:        osResource.GetKernelCommand(),
+		UpdateSources:        osResource.GetUpdateSources(),
+		ImageUrl:             osResource.GetImageUrl(),
+		ImageId:              osResource.GetImageId(),
+		Sha256:               osResource.GetSha256(),
+		ProfileName:          osResource.GetProfileName(),
+		ProfileVersion:       osResource.GetProfileVersion(),
+		InstalledPackagesUrl: osResource.GetInstalledPackagesUrl(),
+		SecurityFeature:      inv_osv1.SecurityFeature(osResource.GetSecurityFeature()),
+		OsType:               inv_osv1.OsType(osResource.GetOsType()),
+		OsProvider:           inv_osv1.OsProviderKind(osResource.GetOsProvider()),
+		Description:          osResource.GetDescription(),
+		Metadata:             osResource.GetMetadata(),
+		ExistingCvesUrl:      osResource.GetExistingCvesUrl(),
+		ExistingCves:         osResource.GetExistingCves(),
+		FixedCvesUrl:         osResource.GetFixedCvesUrl(),
+		FixedCves:            osResource.GetFixedCves(),
```

Comment:
Actually, realized now. `existingCves` and `fixedCves` are read-only filds too from API perspective. No need to convert them here.